src/api/protectedApiClient.ts

```diff
@@ -82,6 +89,15 @@ const registerTickets = (request: RegisterTicketsRequest) => {
   });
 };
 
+const updateTickets = (eventId: number, request: UpdateTicketsRequest) => {
+  return AppAxiosInstance.put(
+    `${ProtectedApiClientRoutes.REGISTER_TICKETS}/${eventId}`,
+    request,
+  ).then((res) => {
+    return;
+  });
```

Comment:
❓ huh

Reply:
I think this was from copy pasting... return type is void I think? we can resolve to undefined instead